src/components/containers/menu/MenuPanel.js

```diff
@@ -26,30 +24,24 @@ const StyledChildrenContainer = styled.div`
 `;
 
 const MenuPanel = (props, context) => {
-  const { children, isOpen, closeFunction } = props;
-
+  const { children, isOpen, onClose } = props;
   return (
-    <ThemeProvider theme={defaultTheme}>
-      <StyledPanel isOpen={isOpen}>
-        <Spacer>
-          <StyledDismissButton onClick={closeFunction} />
-        </Spacer>
-        <StyledChildrenContainer inline={context.inline}>
-          {children}
-        </StyledChildrenContainer>
-      </StyledPanel>
-    </ThemeProvider>
+    <StyledPanel isOpen={isOpen} className="menu-panel" t>
```

Comment:
You've got a little "t" at the end here. :)

Reply:
Yuck removing now